docs/docs/recipes.md

```diff
@@ -152,6 +152,34 @@ By targeting the HTML `body` element, your font will apply to most text on the p
 - More on [importing assets into files](/docs/importing-assets-into-files/]
 - [Using Typography.js for Google fonts](/docs/typography-js/)
 
+### Using Google Fonts
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start)
+- The [Gatsby CLI](/docs/gatsby-cli/) installed
+- Choosing a font package from [the typefaces project](https://github.com/KyleAMathews/typefaces)
+
+#### Directions
+
+1. Run `npm install --save yourchosenfont`, replacing `yourchosenfont` with the name of the font you want to install from [the typefaces project](https://github.com/KyleAMathews/typefaces).
+2. Add `import "yourchosenfont"` to a layout template, page component, or `gatsby-browser.js`.
+3. Once it's imported, you can reference the font in a CSS stylesheet.
+
+```css:title=src/components/layout.css
+body {
+    font-family: yourchosenfont;
+} 
+```
+
+Since you are now hosting your own Google font within your project, there is no need to pull it in when your site loads, increasing your site's speedindex by up to ~300 miliseconds on desktop and 1+ seconds on 3g!
```

Comment:
Now that I'm looking at this again, I think this would make a great introductory sentence before the prerequisites. How about something like:

> Hosting your own Google fonts locally within a project means they won't have to be fetched over the network when your site loads...

What do you think?